src/GitVersion.Core.Tests/VersionCalculation/Strategies/ConfigNextVersionBaseVersionStrategyTests.cs

```diff
@@ -7,7 +7,7 @@
 namespace GitVersion.Core.Tests.VersionCalculation.Strategies;
 
 [TestFixture]
-public class ConfigNextVersionBaseVersionStrategyTests : TestBase
+public class ConfiguredNextVersionBaseVersionStrategyTests : TestBase
```

Comment:
It seems like the class name here contains `BaseVersion` for no particular reason?
```suggestion
public class ConfiguredNextVersionStrategyTests : TestBase
```

Reply:
done